src/utils/get-sorted-nodes-by-import-order.ts

```diff
@@ -70,8 +70,13 @@ export const getSortedNodesByImportOrder: GetSortedNodes = (nodes, options) => {
     for (const group of importOrder) {
         // If it's a custom separator, all we need to do is add a newline
         if (isCustomGroupSeparator(group)) {
+            const lastNode = finalNodes[finalNodes.length - 1];
+            // Avoid empty new line if first group is empty
+            if (!lastNode) {
```

Comment:
This feels a little bit indirect.  Could we check if `finalNodes.length === 0`? Would that accomplish the same thing?

Reply:
It would be the same thing for this first checks, but then I would need to assert TS that the list is non empty for the next check (or add a useless runtime check)